### PR TITLE
bugfix/BS-2993-align-ipaddress

### PR DESCRIPTION
**Pull request by kfirbenhaim**

### CODE REVIEW

File: src/main/java/security/build/pdp/request/PDPRequestInput.java [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import org.springframework.boot.jackson.JsonComponent;
 
 @JsonComponent
 public class PDPRequestInput {
     PDPRequestIncomingHttp request;
     PDPRequestResources resources;
-    String source;
+    ObjectNode source;
```
Comment (amirbenun) [inline src/main/java/security/build/pdp/request/PDPRequestInput.java:11]: Create a class `ConnectionTuple` (or whatever) that has `ipAddress` and `port`.
`PDPRequestInput.destination` should also be of type `ConnectionTuple`.

**Review by amirbenun — CHANGES_REQUESTED** [review]

File: src/main/java/security/build/pdp/request/PDPRequestInput.java [hunk 2]
```diff
@@ -6,26 +6,26 @@
...
+    PDPConnectionTuple source;
+    PDPConnectionTuple destination;
 
     public PDPRequestInput(PDPRequestIncomingHttp request, PDPRequestResources resources, String source, String destination) {
         this.request = request;
         this.resources = resources;
-        this.source = source;
-        this.destination = destination;
+        this.source = new PDPConnectionTuple(source, 0);
+        this.destination = new PDPConnectionTuple(destination, 0);
```
Comment (amirbenun) [inline src/main/java/security/build/pdp/request/PDPRequestInput.java:16]: Make sure you fix it on a different pr

**Review by amirbenun — APPROVED** [review]

File: src/main/java/security/build/pdp/request/PDPRequestInput.java (hunk 2 above)
Comment (kfirbenhaim) [reply src/main/java/security/build/pdp/request/PDPRequestInput.java:16]: https://build-security.atlassian.net/browse/BS-2993
